docs/notebooks/algorithms/fit_spline_skeletons_tutorial.ipynb

```diff
@@ -0,0 +1,153 @@
+{
```

Comment:
There's a lot of commented out code, can we delete it if it's not necessary?

Also I get an error on plt.draw() - AttributeError: 'LineCollection' object has no attribute 'do_3d_projection'